.circleci/config.yml

```diff
@@ -15,7 +15,7 @@
  version: 2.1
  workflows:
    pull-request:
-     unless:
+     equal:
        equal: ["trunk", << pipeline.git.branch >>]
```

Comment:
Won't this run on master now?

Reply:
Yes, this is on purpose. I can't figure out the deploy to ECS

Reply:
I'm going to revert when I'm done testing lol

Reply:
Ah. Usually my team won't switch the entire workflow as it's higher risk, rather temporarily insert the needed step into the PR workflow. But nothing is gonna blow up right now.